industrial_ci/scripts/rerun_ci

```diff
@@ -52,26 +52,26 @@ fi
 repo_dir=$(cd "$1" && pwd)
 shift
 
-_COMMIT_IMAGE_MSG="$repo_dir $*"
-env_hash=($(sha256sum <<< "$_COMMIT_IMAGE_MSG"))
-_COMMIT_IMAGE="industrial-ci/rerun_ci/$(basename "$repo_dir"):${env_hash:0:12}"
+if [ -z $DOCKER_COMMIT_MSG ]; then DOCKER_COMMIT_MSG="$repo_dir $*"; fi
```

Comment:
`DOCKER_COMMIT_MSG` and `DOCKER_COMMIT` should not get overwritten from the outside, they are crucial to the argument hashing.
Please remove these if clauses.